### ISSUE TITLE
read receipts get covered by room status bar when last message in room is a picture

### ISSUE THREAD

**Issue by bwindels** [vector-im/riot-web#12675]

**Comment by bwindels** [comment]
It happens for text messages too actually.

**Comment by turt2live** [comment]
Likely to be highly related to https://github.com/vector-im/riot-web/issues/12660

**Comment by turt2live** [comment]
@bwindels I can't reproduce this. Do you have more specific instructions?

**Comment by bwindels** [comment]
> @bwindels I can't reproduce this. Do you have more specific instructions?

Yeah, looks like #12660, when hovering the read receipt in the screenshot below, it highlights the event above of it.

![image](https://user-images.githubusercontent.com/274386/76333728-f117eb00-62e9-11ea-9a3b-fcf9d6c8be31.png)

I was able to reproduce the read receipt being covered when in a test room with two users and they are both typing.

**Comment by bwindels** [comment]
Here's a GIF of reproducing it. It happens when the other user's "is typing" notification disappears and their message arrives. Then, after reproing, I'm (slowly and confusingly) trying to show that the RR avatar is still rendered but outside of the timeline viewport. 

![rrdisappearingafterstoptyping2](https://user-images.githubusercontent.com/274386/76334187-99c64a80-62ea-11ea-9483-2daa46fa1bd8.gif)

**Comment by turt2live** [comment]
![image](https://user-images.githubusercontent.com/1190097/76334864-4daee200-62b9-11ea-82b9-c6e81387c8a2.png)

:( 

Browser and OS?

**Comment by bwindels** [comment]
Firefox 73.0.1 (64-bit) on Fedora

**Comment by uhoreg** [comment]
I see the issue too.  Firefox 73.0.1 on Debian.

**Comment by turt2live** [comment]
Other reports are from MacOS, chrome. Seems like a not-windows issue or I'm failing to follow instructions.

I could not reproduce this on Fedora/Firefox or Ubuntu/Firefox.

We'll try reverting https://github.com/matrix-org/matrix-react-sdk/pull/3892 and see how it goes for now.

**Comment by turt2live** [comment]
Revert has been pushed, putting this into the infinite maintenance backlog.
